README.md

```diff
@@ -1,106 +1,275 @@
+* When the user clicks one of the **Buy Now** buttons, the demo invokes `track`, passing in the event key `item_purchase` and the user ID to track the event.
 
-* View the reference [documentation](http://developers.optimizely.com/server/reference/index.html?language=javascript).
+## Prerequisites
+* [Optimizely account](https://app.optimizely.com/signin)
+* [npm](https://www.npmjs.com/get-npm)
 
-* Latest [JavaScript SDK](https://github.com/optimizely/javascript-sdk)
+## Quick start
+This section shows you how to prepare the project in the Optimizely portal and then run the demo in a browser hosted by a local server.
 
-## Demo App
+### Preparing an Optimizely Project
+This section provides the steps to prepare an Optimizely project in your dashboard.
 
-This example app illustrates how an online retailer could developer a new feature behind a toggle, rollout it out gradually to more users and run an experiment on the new vs old experience by tracking a business metric.
+1. Clone or download the **javascript-sdk-demo-app** package.
+2. Log in or create an [Optimizely Account](https://app.optimizely.com/signin).
+3. Create a project via the Optimizely dashboard.
+4. Add a Feature with the key `sorting_enabled`. This will act as a toggle for the sorting Feature.
+5. Add a Variable Key to the Feature called `welcome_message`, set a default value, and save the Feature.
+6. Add an Event Key called `item_purchase` and save the event. This event will be triggered when one of the **Buy** buttons is clicked.
+7. Create a new Experiment using the Feature `sorting_enabled`. Create two variations with a 50/50 split where only one variation is enabled for the sorting Feature.
+8. Navigate to the directory where the package was downloaded to in Step 1 and open **./constants.js** in a text editor.
+9. Update the default `datafileURL` field in the file with the URL from your Optimizely dashboard, located under **Settings** > **Environments**. The URL points to the JSON data file stored on Optimizely that will be used by code within the demo.
+```javascript
+// constants.js
 
-Using the instructions below, you can run the app locally and mimic bucketing website visitors by entering unique user IDs into the input bar. For example, the user ID “Matt” would simulate a unique visitor and the SDK would determine whether the feature shold be shown. The bucket that is given to a specific unique visitor, such as Matt, will be deterministic. This means as long as the Optimizely conditions remain the same, Matt will always get the same experience.
- 
-<img src="https://github.com/optimizely/javascript-sdk-demo-app/blob/master/src/images/screenshot.png" width="450">
+// Default datafile provided to get up and running quickly. Replace with your own!
+//const datafileURL = 'https://cdn.optimizely.com/datafiles/CpqC7ch493FEqa3HzY3963.json';
```

Comment:
This looks like an actual URL. Would recommend changing it to something random which doesn't have anything associated with anyone's account like `https://cdn.optimizely.com/datafiles/AbCdE12345.json`